spec/lib/import_users_spec.rb

```diff
@@ -25,12 +25,12 @@
   end
 
   it 'creates users from a csv file' do
-    headers = [:row_number, :username, :password_digest, :title, :first_name, :last_name, :full_name, :email_address]
```

Comment:
We could put `full_name` back in the importer but would need a way to split out suffixes.